src/TestRunner/TestRunner.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I have to admit that I'm not familiar with the project structure of the repo, but seing a "runner" project targeting netstandard confuses me a bit.

Reply:
It is a lib for the base runner